graph/src/env/mod.rs

```diff
@@ -212,6 +212,24 @@ pub struct EnvVars {
     /// Set the maximum grpc decode size(in MB) for firehose BlockIngestor connections.
     /// Defaults to 25MB
     pub firehose_grpc_max_decode_size_mb: usize,
+
+    /// Sets the token that is used to authenticate graphman GraphQL queries.
+    ///
+    /// The authentication token must be exactly 64 characters long.
+    /// This is equivalent to an SHA-256 hash represented as a hex string.
+    ///
+    /// If not specified, the graphman server will not start.
+    pub graphman_server_auth_token: Option<String>,
+
+    /// Set the maximum query depth for the graphman GraphQL server.
+    ///
+    /// The default value is 255.
+    pub graphman_server_query_max_depth: u8,
+
+    /// Set the maximum query complexity for the graphman GraphQL server.
+    ///
+    /// The default value is 1,000,000.
+    pub graphman_server_query_max_complexity: u32,
```

Comment:
We've tried to use query complexity for the 'normal' GraphQL API, and found it's pretty much useless in determining how hard a query is. Do we really need a setting for max complexity at all?

Reply:
Removed these configuration options for now